### ISSUE TITLE
[Bug ]ChatQnA - compose.yaml for Gaudi - Habana devices

### ISSUE THREAD

**Issue by pallavijaini0525** [opea-project/GenAIExamples#739]
### Priority

Undecided

### OS type

Ubuntu

### Hardware type

Gaudi2

### Installation method

- [X] Pull docker images from hub.docker.com
- [ ] Build docker images from source

### Deploy method

- [X] Docker compose
- [ ] Docker
- [ ] Kubernetes
- [ ] Helm

### Running nodes

Single Node

### What's the version?

https://github.com/opea-project/GenAIExamples/blob/main/ChatQnA/docker/gaudi/compose.yaml

### Description

For the ChatQnA application,  https://github.com/opea-project/GenAIExamples/blob/main/ChatQnA/docker/gaudi/compose.yaml

compose.yaml has two containers where both are requesting HABANA_VISIBLE_DEVICES=all, For multi tenancy we need to specify the device ids instead of all, 

with the existing compose.yaml, error is as below.

Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error running hook #0: error running hook: exit status 1, stdout: , stderr: exposing interfaces: failed creating temporary link on host: invalid argument

### Reproduce steps

Run the docker compose file - https://github.com/opea-project/GenAIExamples/blob/main/ChatQnA/docker/gaudi/compose.yaml after setting the env variables specified in https://github.com/opea-project/GenAIExamples/tree/main/ChatQnA/docker/gaudi#setup-environment-variables

**Comment by lvliang-intel** [comment]
"all" means the system will allocate the device automatically. Users don't need to set the device number.

**Comment by lvliang-intel** [comment]
Yes, the system will automatically allocate a Gaudi card. Allowing users to specify the card number may not be a good idea. Normal users have no more knowledge about the Gaudi system.

**Comment by pallavijaini0525** [comment]
yes please
